pom.xml

```diff
@@ -1588,34 +1588,40 @@
                     <executions>
                         <execution>
                             <id>verify-forbidden-apis</id>
-                                <configuration>
-                                     <targetVersion>${maven.compiler.argument.target}</targetVersion>
-                                     <!-- if the used Java version is too new, don't fail, just do nothing: -->
-                                     <failOnUnsupportedJava>false</failOnUnsupportedJava>
-                                     <excludes>
-                                         <!-- JBoss Logger formats messages according to default Locale (bug or feature?) -->
-                                         <exclude>**/*_$logger.class</exclude>
-                                         <!-- JBoss message bundles format messages -->
-                                         <exclude>**/*_$bundle.class</exclude>
-                                     </excludes>
-                                     <bundledSignatures>
-                                         <!-- This will automatically choose the right signatures based on 'targetVersion': -->
-                                         <bundledSignature>jdk-unsafe</bundledSignature>
-                                         <bundledSignature>jdk-deprecated</bundledSignature>
-                                         <bundledSignature>jdk-system-out</bundledSignature>
-                                         <bundledSignature>jdk-non-portable</bundledSignature>
-                                         <bundledSignature>jdk-internal</bundledSignature>
-                                     </bundledSignatures>
-                                     <failOnMissingClasses>false</failOnMissingClasses>
-                                     <failOnUnresolvableSignatures>false</failOnUnresolvableSignatures>
-                                     <!--
-                                     The method SharedSessionContract#getTransaction() should never be used
-                                     as it is only reliable in certain configurations.
-                                      -->
-                                     <signatures>
-org.hibernate.SharedSessionContract#getTransaction()
-                                     </signatures>
-                                 </configuration>
+                            <configuration>
+                                 <targetVersion>10</targetVersion>
```

Comment:
Isn't there a risk that we introduce calls to APIs introduced in Java 10?

Reply:
It's the other way around: it's about banning calls, not introducing calls.

Essentially the sets of deprecated/removed methods included in ForbiddenAPI for JDK10 are more up to date than those in JDK9.

Sadly the JDK11 list isn't included yet, but at least if we avoid anything which is deprecated in JDK10 changes are good that it will be fine in JDK11 as well.